drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -58,6 +58,13 @@
 #include "vc4_hdmi_regs.h"
 #include "vc4_regs.h"
 
+/* "Broadcast RGB" property.
```

Comment:
The preferred format for a multiline comments is to have the `/*` and `*/` on lines of their own.